src/components/ProductSwitcher.js

```diff
@@ -45,16 +50,20 @@ class ProductSwitcher extends React.Component {
     });
   }
 
+  productName() {
+    return this.props.product_name ? this.props.product_name : this.props.product
```

Comment:
This could be just stored in a variable. Don't need to create a function.

Reply:
Good one!